core/src/main/resources/jenkins/model/Jenkins/manage.jelly

```diff
@@ -29,7 +29,6 @@ THE SOFTWARE.
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
 <l:layout title="${%Manage Jenkins}" permissions="${app.MANAGE_AND_SYSTEM_READ}">
   <l:header>
-     <link rel="stylesheet" href="${resURL}/bootstrap/css/bootstrap.min.css" type="text/css" />
```

Comment:
Is this related to fonts changes?

Reply:
Yep, bootstrap 3 was not actually being used and it has risks of messing with default font settings.